batchflow/models/torch/base.py

```diff
@@ -1847,3 +1850,60 @@ def _parse_profilers(self):
                                          columns=['ncalls', 'CPU_tottime', 'CPU_cumtime', 'CUDA_cumtime'])
         self.profile_info['CPU_tottime_avg'] = self.profile_info['CPU_tottime'] / self.profile_info['ncalls']
         self.profile_info['CUDA_cumtime_avg'] = self.profile_info['CUDA_cumtime'] / self.profile_info['ncalls']
+
+
+    # Utilities for activations
+    @staticmethod
+    def get_blocks_and_activations(model, modules=None):
+        """ Retrieve intermediate blocks of the neural network model
+        and corresponding activation names.
+
+        Parameters
+        ----------
+        model : Network
+        modules : str or list of str, default None
+            The main parts of the model for retrieving activations from.
+            If None, all of the parts from the model.config['order'] wil be used.
+
+        Returns
+        -------
+        activation_blocks : list of str
+            List with activation blocks which will be passed into `outputs` parameter of `self.predict`.
```

Comment:
Description must contain information only in context of the current object (method  `get_blocks_and_activations`).